example/fastapi_sample_app.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+import fastapi
+
+import json_logging
+
+app = fastapi.FastAPI()
+
+# init the logger as usual
+logger = logging.getLogger(__name__)
```

Comment:
you're missing the **logger.setLevel** call

Reply:
This is actually not necessary. All what's needed is configured further down with a config dict.
I'm doing it that way, so that I can pass this configuration to uvicorn which configures logging itself. When I tried it locally in the script as for the other backends it didn't work out.

Reply:
a bit weird, when i run the sample directly from terminal the logging statement didnt emit anything, only works with setLevel call

Reply:
Ah, when you copy the code directly into an IPython shell, you mean? I also tested it and indeed that didn't work without `logger.setLevel()`. I only tested to run the script as dedicated python process (`python fastapi_sample_app.py`).

But right, with an additional `logger.SetLevel()` it works under all circumstances, so I just added that.